scrapers_next/in/people.py

```diff
@@ -46,30 +47,36 @@ def process_page(self):
         )
 
         social_dict = {
-            "facebook": None,
-            "instagram": None,
-            "twitter": None,
-            "youtube": None,
+            "facebook": [],
+            "instagram": [],
+            "twitter": [],
+            "youtube": [],
         }
+
+        for soc in social_dict.keys():
+            soc_pattern = re.compile(rf"(.+)({soc}\.com/)(.+)")
+            social_dict[soc].append(soc_pattern)
+
         social_links = CSS("div .fusion-social-links a").match(self.root)
         for link in social_links:
             href = link.get("href").lower()
             for soc in social_dict.keys():
                 if soc in href:
-                    raw_handle = re.search(rf"(.+)({soc}\.com/)(.+)", href).groups()[-1]
+                    pattern = social_dict[soc][0]
+                    raw_handle = pattern.search(href).groups()[-1]
                     handle = re.sub("/", "", raw_handle)
-                    social_dict[soc] = handle
+                    social_dict[soc].append(handle)
 
         p.capitol_office.address = addr
 
-        if social_dict["facebook"]:
-            p.ids.facebook = social_dict["facebook"]
-        if social_dict["instagram"]:
-            p.ids.instagram = social_dict["instagram"]
-        if social_dict["twitter"]:
-            p.ids.twitter = social_dict["twitter"]
-        if social_dict["youtube"]:
-            p.ids.youtube = social_dict["youtube"]
+        if len(social_dict["facebook"]) > 1:
```

Comment:
This is a bit confusing. Will there always be at least one value in each key? Shouldn't they be empty unless you find something?

Reply:
Totally. I was handling it all with one dictionary with `"social media": []` where the first element of the list was always the precompiled regex pattern for that social media platform, and the second element would user handle for that social media platform (if encountered).

But I went ahead in my newest commit and separated things out on lines 51-53 into:
- A list of the social media we're looking for
- A dictionary to store any handles we find
- A dictionary to store the regex patterns

With this change,[ the code is much simpler / more pythonic now](https://github.com/openstates/openstates-scrapers/pull/4233/commits/5f77c3a7c225632f85267946108f8646614db4a8#diff-9eae9c00f2832ba4d89c6294f928772d6d554299758b67c70db01777d9283412R51-R60).